lib/umfmessage.js

```diff
@@ -152,41 +150,41 @@ function createMessageInstance(message) {
       return true;
     }
   });
-  if (message.to) {
+  if (isNotEmptyString(message.to)) {
     proxy.to = message.to;
   }
-  if (message.from || message.frm) {
-    proxy.from = message.from || message.frm;
+  if (isNotEmptyString(message.from ?? message.frm)) {
```

Comment:
Looks like support for ES2020's Nullish Coalescing Operator only arrived in Node v14.  This would be a breaking change for anyone using less than Node v14?

Reply:
Yeah, I am so used using Typescript and then compiling to like es5 so I don't think about such things. I'll add a function that works like the operator and use that instead as soon as I am back home, but then I'll have to do that for hydra-cli too.